File: a414位元運算/compute.java
import java.util.*;

public class JAVA
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

        while(true)
        {
            int n = sc.nextInt();
            if(n == 0)
            {
                break;
            }
            int cnt = 0;
            while((n&1) == 1)
            {
                cnt++;
                n >>= 1;
            }
            System.out.println(cnt);
        }
    }
}

File: a414位元運算/compute.py
while True:
    try:
        n = int(input())
    except EOFError:
        break
    
    if n == 0:
        break
    
    cnt = 0
    while n != 0:
        r = n % 2
        if r != 1:
            break
        cnt += 1
        n >>= 1
    print(f'{cnt}')
